### ISSUE TITLE
Literal text "UtimesNano" embedded in error messages

### ISSUE THREAD

**Issue by cowwoc** [restic/restic#5133]
Output of `restic version`
--------------------------
restic 0.17.3 compiled with go1.23.3 on windows/amd64

What backend/service did you use to store the repository?
---------------------------------------------------------
b2

Problem description / Steps to reproduce
----------------------------------------
When restic outputs an error (e.g. `ignoring error for \C\users: UtimesNano: Access is denied.`) it contains the literal text `UtimesNano` which seems like a bug. I assume this text is meant to be replaced by a timestamp? If you search other bug reports by Windows users, you will spot this text showing up all over the place.

**Comment by MichaelEischer** [comment]
> text `UtimesNano` which seems like a bug.

No, the output is correct, although not particularly descriptive from a user perspective. What happened is that restic failed to set the timestamp for the given file `\C\users` (whatever that maps to in the target folder) due to an `Access is denied` error. `UtimesNano` is the go/restic-internal name of the operation used to set timestamps. (`utimes` is the syscall name to set timestamps on unix and `nano` signals that nanosecond precision is supported)

> If you search other bug reports by Windows users, you will spot this text showing up all over the place.

Which search query did you use exactly? https://github.com/restic/restic/issues?q=is%3Aissue+%22UtimesNano%22+ basically only returns issues that have been resolved years ago.

**Comment by cowwoc** [comment]
Ah, okay. I'll go ahead and close this issue then.

And to your point, it would probably be clearer if "UtimesNano" was replaced by "Unable to update file timestamp".

**Comment by MichaelEischer** [comment]
https://github.com/restic/restic/pull/5134 makes the error message more descriptive.

**Comment by cowwoc** [comment]
Thank you
